README.md

```diff
@@ -124,11 +130,12 @@ initialization scripts (and later daemons) to pick from.
 ovs-vsctl set Open_vSwitch . external_ids:k8s-api-server="0.0.0.0:8080"
 ```
 
-Install the following additional packages for python.
+Clone this repository and install the executables.
 
 ```
-pip install netaddr
-pip install eventlet
+git clone https://github.com/openvswitch/ovn-kubernetes
+cd ovn-kubernetes
+pip install -e `pwd`
```

Comment:
Your statement made me re-try on 'pip install' and 'pip uninstall'. I had mixed up usage of setup.py directly with pip and the only thing that was working for me was 'pip install -e'. Now that I understand this better, it makes sense to just say 'pip install .' in documentation.